Automatic Validation ended with:
> 2024-01-27T06:45:47.4996335Z ##[error] Manifest Error: DisplayVersion declared in the manifest has overlap with existing DisplayVersion range in the index. Existing DisplayVersion range in index:  [ [VERSION, VERSION]]

(Automated response - build 669.)

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.installer.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.installer.yaml
@@ -23,27 +23,5 @@ Installers:
     PortableCommandAlias: cf8
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli_8.7.2_win32.zip
   InstallerSha256: 3125EF78DEF460D4A9C8C5DEACAF4F291F10BF484E9C96D493F0FFA7DE30B5BF
-- Architecture: x64
-  NestedInstallerType: inno
-  NestedInstallerFiles:
-  - RelativeFilePath: cf8_installer.exe
-  Scope: machine
-  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_winx64.zip
-  InstallerSha256: 489BBBF3DB174195C430E23F4A96BC27951AE8FE54159BF5042B431180F332AD
-  AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.2
-    ProductCode: Cloud Foundry CLI_is1
-  ElevationRequirement: elevatesSelf
-- Architecture: x86
-  NestedInstallerType: inno
-  NestedInstallerFiles:
-  - RelativeFilePath: cf8_installer.exe
-  Scope: machine
-  InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_win32.zip
-  InstallerSha256: 6CECF7ADC3C8BC7A324B536B709D70A5DF51309F14EE076DD64C6A7D512F94AE
-  AppsAndFeaturesEntries:
-  - DisplayVersion: v8.7.2
-    ProductCode: Cloud Foundry CLI_is1
-  ElevationRequirement: elevatesSelf
 ManifestType: installer
 ManifestVersion: 1.5.0

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.installer.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.installer.yaml
@@ -31,7 +31,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_winx64.zip
   InstallerSha256: 489BBBF3DB174195C430E23F4A96BC27951AE8FE54159BF5042B431180F332AD
   AppsAndFeaturesEntries:
-  - DisplayVersion: VERSION
+  - DisplayVersion: v8.7.2
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 - Architecture: x86
@@ -42,7 +42,7 @@ Installers:
   InstallerUrl: https://s3-us-west-1.amazonaws.com/v8-cf-cli-releases/releases/v8.7.2/cf8-cli-installer_8.7.2_win32.zip
   InstallerSha256: 6CECF7ADC3C8BC7A324B536B709D70A5DF51309F14EE076DD64C6A7D512F94AE
   AppsAndFeaturesEntries:
-  - DisplayVersion: VERSION
+  - DisplayVersion: v8.7.2
     ProductCode: Cloud Foundry CLI_is1
   ElevationRequirement: elevatesSelf
 ManifestType: installer

--- a/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.locale.en-US.yaml
+++ b/manifests/c/CloudFoundry/CLI/v8/8.7.2/CloudFoundry.CLI.v8.locale.en-US.yaml
@@ -9,7 +9,7 @@ PublisherUrl: https://cli.cloudfoundry.org/
 PublisherSupportUrl: https://github.com/cloudfoundry/cli/issues
 # PrivacyUrl: 
 # Author: 
-PackageName: Cloud Foundry CLI
+PackageName: Cloud Foundry CLI v8
 PackageUrl: https://cli.cloudfoundry.org/en-US/v8/
 License: Apache-2.0
 LicenseUrl: https://github.com/cloudfoundry/cli/blob/main/LICENSE

